assets/js/slideshow.js

```diff
@@ -0,0 +1,28 @@
+let index = 0;
+showSlides();
+
+function showSlides() {
+    let i;
```

Comment:
Best to keep `i` just in the for loops so that the scope doesn't accidentally mess something up on the outside